Task_2/mlcubes/model/README.md

```diff
@@ -0,0 +1,209 @@
+│       └── output           # Location where inference outputs are stored.
+└── project
+    ├── Dockerfile           # Docker file with instructions to create the image.
+    ├── mlcube.py            # Python entrypoint used by MLCube, contains the logic for MLCube tasks.
+    ├── model_ckpts          # Folder with checkpoint files loaded for inference.
+    ├── parameters.yaml      # File with parameters used by inference procedure.
+    ├── requirements.txt     # Python requirements needed to run the project inside Docker.
+    └── src
+        ├── my_logic.py      # Python file that contains the main logic of the project.
+        └── utils
+            └── utilities.py # Python utilities file that stores useful functions.
+```
+
+## How to modify this project
+
+You can change each file described above in order to add your own implementation. In case you need more information on the internals of MLCube, check out the official [git repository](https://github.com/mlcommons/mlcube) or [documentation](https://mlcommons.github.io/mlcube/).
+
+<details><summary><b>Requirements file </b></summary>
+<p>
+
+In this file (`requirements.txt`) you can add all the python dependencies needed for running your implementation. These dependencies will be installed during the creation of the docker image, which happens automatically when you run the ```mlcube run ...``` command.
+</p>
+</details>
+
+<details><summary><b>Dockerfile </b></summary>
+<p>
+
+This file can be adapted to add your own docker labels, install some OS dependencies or to change the base docker image. Note however that we *strongly recommend* to use one of our proposed base images (`nvcr.io/nvidia/pytorch:20.08-py3` or tensorflow equivalent), to make sure your application can be executed in the federated evaluation. Note that the [pytorch (or tensorflow) version](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) inside this container is 1.7.0 (or 2.2.0), so for inference you may not be able to use features introduced in later versions, unfortunately.
+
+</p>
+</details>
+
+<details><summary><b>MLCube yaml file </b></summary>
+<p>
+
+`mlcube.yaml` contains instructions about the docker image and platform that will be used, information about the project (name, description, authors), and also the tasks defined for the project. **Note** that this file is not submitted and changes will hence not have any effect in the official evaluation. We will use the provided template with the name of your docker image instead.
+
+In the existing implementation you will find the `infer` task, which will be executed in the federated evaluation. It takes the following parameters:
+
+- Input parameters:
+  - data_path: folder path containing input data
+  - checkpoint_path: folder path containing model checkpoints
+  - parameters_file: Extra parameters
+- Output parameters:
+  - output_folder: folder path where output data will be stored
```

Comment:
Small typo here. It should be output_path, not output_folder. Just so the competitors don't get confused here